client/common/cmdline.c

```diff
@@ -48,6 +49,7 @@
 
 #include <freerdp/client/cmdline.h>
 #include <freerdp/version.h>
+#include <freerdp/utils/smartcard_cli.h>
```

Comment:
Moving `smartcard_cli` to `client` library should also be represented in header location. Maybe `freerdp/client/utils`?